gap_linux.go

```diff
@@ -496,3 +533,65 @@ func (a *Adapter) SetRandomAddress(mac MAC) error {
 
 	return nil
 }
+
+// SetAdapterAlias sets the alias of the adapter in BlueZ.
+func (a *Adapter) SetAdapterAlias(alias string) error {
```

Comment:
Does this function really need to be exported? Since there is no equivalent on other platforms, we try to avoid it. Since it is not used in the PR itself, is this something you need for your specific application?

Reply:
> In gap_linux.go I could modify Advertisement.Configure to use the LocalName to also set the adapter Alias. Since this is host stack specific, I don't think other changes (e.g. adding AdvertisementOptions.AliasName) makes sense, so if there are any changes I suggest keeping changes internal to the linux stack.

That seems like a reasonable approach.